lib/fauxpaas/pipeline/pipeline.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Fauxpaas
+  module Pipeline
+
+    # Pipelines represent the highest level processes that the application
+    # performs. Pipelines tie everything together.
+    class Pipeline
+      extend Forwardable
+
+      def_delegators :@command, :instance, :user, :logger
```

Comment:
I think environmental/system concerns like the logger should probably be managed by the pipeline itself, rather than the command, which I see as a straightforward serialization of a user request. This is along the lines of my other comment that I see the pipeline as the execution environment (which itself can reach out to the service locator).